rust/lance/src/dataset/scanner.rs

```diff
@@ -338,6 +390,21 @@ impl Scanner {
         self
     }
 
+    pub fn order_by(&mut self, ordering: Option<Ordering>) -> Result<&mut Self> {
```

Comment:
Only supporting one column for now?

Reply:
I've updated this to allow for multi-column sorts.